app/src/main/java/com/willowtree/vocable/utils/VocableSharedPreferences.kt

```diff
@@ -38,4 +39,12 @@ class VocableSharedPreferences : KoinComponent {
             encryptedPrefs.edit().putStringSet(KEY_MY_SAYINGS, sayings).apply()
         }
     }
+
+    fun setHeadTrackingEnabled(enabled: Boolean) {
+        encryptedPrefs.edit().putBoolean(KEY_HEAD_TRACKING_ENABLED, enabled).apply()
+    }
+
+    fun getHeadTrackingEnabled(): Boolean {
```

Comment:
Could just make this `fun getHeadTrackingEnabled() = encryptedPrefs.getBoolean(KEY_HEAD_TRACKING_ENABLED, true)`?